src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {isMarried
+        ? `My ${sex === 'm' ? "wife's" : "husband's"} name is ${partnerName}`
+        : `I am not married`}
```

Comment:
It's better to create separate variable before return statement where you can store the result of inner conditional statement (checking sex === 'm')
```suggestion
      {isMarried
        ? `My ${partner}'s name is ${partnerName}`
        : `I am not married`}
```